lms/static/js/spec/navigation_spec.js

```diff
@@ -0,0 +1,93 @@
+define(['jquery', 'js/utils/navigation'], function($) {
+    'use strict';
+
+    describe('Course Navigation Accordion', function() {
+        var accordion, button, heading, chapterContent, chapterMenu;
+
+        beforeEach(function() {
+            loadFixtures('js/fixtures/accordion.html');
+
+            accordion = $('.accordion');
+            button = accordion.children('.button-chapter');
+            heading = button.children('.group-heading');
+            chapterContent = accordion.children('.chapter-content-container');
+            chapterMenu = chapterContent.children('.chapter-menu');
+
+            spyOn($.fn, 'focus').andCallThrough();
+            edx.util.navigation.init();
+        });
+
+        describe('constructor', function() {
+
+            describe('always', function() {
+
+                it('ensures accordion is present', function() {
+                    expect(accordion.length).toBe(1);
+                });
+
+                it('ensures aria attributes are present', function() {
+                    expect(chapterContent).toHaveAttr({
+                        'aria-expanded': 'true'
+                    });
+                });
+
+                it('ensures only one active item', function() {
+                    expect(chapterMenu.find('.active').length).toBe(1);
+                });
+            });
+
+            describe('open section with mouse click', function() {
+
+                it('ensures new section is opened and previous section is closed', function() {
+                    button:eq(1).click();
+
+                    expect(chapterContent:eq(0)).not.toHaveClass('is-open');
```

Comment:
@clrux @ahsan-ul-haq This code is not valid JavaScript, so the test is silently failing. It looks like this was CoffeeScript that didn't get converted correctly.

@benpatterson Any idea why the JSHint wouldn't have found this? It would also be good to get js-test-tools to fail for invalid test files, rather than silently ignoring them.